recipes/pisces/meta.yaml

```diff
@@ -29,3 +30,9 @@ about:
   home: "https://github.com/Illumina/Pisces"
   license: GPLv3
   summary: "Somatic and germline variant caller for amplicon data. Recommended caller for tumor-only workflows."
+
+
+extra:
+  skip-lints:
+    # dotnet isn't in Conda, so we can't test pisces
+    - missing_tests
```

Comment:
Can you try making mono a dependency and then using it to run tests?

Reply:
Pisces 5.2 targets the dotnet framework only, not mono: https://github.com/Illumina/Pisces#system-requirements

Reply:
Mono is an open source implementation of the dot net framework.

Reply:
Yes but pisces targets dotnet core, not the dotnet framework, so mono won't help.